price-estimator/src/models.rs

```diff
@@ -62,9 +62,9 @@ pub struct EstimatedOrderResult {
     #[serde(with = "display_fromstr")]
     pub quote_token_id: u16,
     #[serde(with = "display_fromstr")]
-    pub buy_amount_in_base: u128,
+    pub buy_amount_in_base: f64,
     #[serde(with = "display_fromstr")]
-    pub sell_amount_in_quote: u128,
+    pub sell_amount_in_quote: f64,
```

Comment:
Does this work with the frontend in the case that atoms=true?

Reply:
Either way all json numbers can be fractions and something that previously wasn't a fraction as u128 still won't be a fraction as f64 so I didn't expect this to be a problem but I'll test it.

Reply:
Tested it without problems.

Reply:
I think, **in a future PR**, we should change these to serialize as numbers instead of strings.

Reply:
Agree, I believe we have an issue for this.